lib/parse.js

```diff
@@ -2174,6 +2256,53 @@ function parse($TEXT, options) {
         });
     };
 
+    function usingdefs(no_in, kind) {
```

Comment:
Now that `UsingDef` has been removed, this helper can be merged with `vardefs` as well.